tests/sentry/incidents/test_logic.py

```diff
@@ -519,63 +508,15 @@ def test_simple(self):
                 'author_email': 'hello@example.com',
                 'author_name': 'Hello',
                 'message': 'i fixed a bug',
-                'patch_set': [
-                    {
-                        'path': 'src/sentry/models/group.py',
-                        'type': 'M',
-                    },
-                ]
-            },
-        ])
-
-        excluded_project = self.create_project()
-        excluded_event = self.store_event(
-            data={
-                'fingerprint': ['group-3'],
-                'message': 'Kaboom!',
-                'platform': 'python',
-                'stacktrace': {
-                    'frames': [
-                        {
-                            "function": "set_commits",
-                            "abs_path": "/usr/src/sentry/src/sentry/models/event.py",
-                            "module": "sentry.models.event",
-                            "in_app": True,
-                            "lineno": 39,
-                            "filename": "sentry/models/event.py",
-                        },
-                    ],
-                },
-                'release': release_2.version,
-            },
-            project_id=excluded_project.id,
-        )
-        excluded_release = self.create_release(project=self.project, version='v9000')
-
-        excluded_commit = 'e' * 40
-        excluded_group = excluded_event.group
-        excluded_release.set_commits([
-            {
-                'id': excluded_commit,
-                'repository': self.repo.name,
-                'author_email': 'hello@example.com',
-                'author_name': 'Hello',
-                'message': 'i fixed a bug',
-                'patch_set': [
-                    {
-                        'path': 'src/sentry/models/event.py',
-                        'type': 'M',
-                    },
-                ]
+                'patch_set': [{'path': 'src/sentry/models/group.py', 'type': 'M'}]
             },
         ])
 
+        commit_ids = Commit.objects.filter(
+            releasecommit__release__in=[release, release_2],
+        ).values_list('id', flat=True)
         incident = self.create_incident(
             self.organization,
-            groups=[group, group_2, excluded_group],
```

Comment:
Is the excluded group case no longer important?

Reply:
Not in this case. The excluded group was basically a factor here when we'd have an excluded project as part of calculation, but now when we precache the suspect commits for incidents we just calculate all suspects regardless of permissions and do the filtering on read instead.